runtime/vm/KeyHashTable.c

```diff
@@ -349,6 +349,12 @@ hashClassTableAt(J9ClassLoader *classLoader, U_8 *className, UDATA classNameLeng
 		if (J9ROMCLASS_IS_HIDDEN(clazz->romClass)) {
 			return NULL;
 		}
+		/* Apply the flags or filters only to frozen classes. */
```

Comment:
This (and the identical comment on line 757) are lacking: Please explain the rules regarding the class flags and the lookup flag.